src/Features/Core/Portable/FindUsages/DefinitionItem.cs

```diff
@@ -156,9 +157,25 @@ internal abstract partial class DefinitionItem
             }
         }
 
+        [Obsolete("Override CanNavigateToAsync instead", error: false)]
         public abstract bool CanNavigateTo(Workspace workspace, CancellationToken cancellationToken);
+        [Obsolete("Override TryNavigateToAsync instead", error: false)]
         public abstract bool TryNavigateTo(Workspace workspace, bool showInPreviewTab, bool activateTab, CancellationToken cancellationToken);
```

Comment:
would like to remove thse, but they are used by typescript.  @DanielRosenwasser can you add appropriate contacts so i can walk you guys through moving to the new async members?